library/conn_bridge.c

```diff
@@ -163,6 +235,16 @@ static void on_output(uv_write_t *wr, int status) {
     free(wr);
 }
 
+static void on_udp_send(uv_udp_send_t *sr, int status) {
+    if (status != 0) {
+        struct ziti_bridge_s *br = sr->handle->data;
+        ZITI_LOG(WARN, "udp_send failed: %d(%s)", status, uv_strerror(status));
```

Comment:
this could be pretty chatty. why not return an error status and propegate back through `on_ziti_data`? I imagine there's some reason, but it's not clear to me so i'm just gonna comment to bring your attention to the thought

Reply:
it won't be chatty because we close bridge on the first failure